.github/workflows/check_milestone.yml

```
@@ -0,0 +1,30 @@
+name: Check Milestone
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  preparerelease:
+    runs-on: ubuntu-latest
```

Should we prevent this from being run on dependency updates? 
```suggestion
  preparerelease:
    runs-on: ubuntu-latest
    if: "!startsWith(github.head_ref, 'latest-dependency-update-')"
```